src/components/Footer/FooterList.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+// import Icon from "../Icon";
+import "./footer.scss";
+export interface FooterProps {
+  title: string;
+  items: {
+    item: string;
+    itemhref?: string;
```

Comment:
Change `itemhref` to `href`. Its obvious.

Reply:
It is done.